app/posts/apply-for-teacher-training/2021-07-14-selecting-references.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Selecting references
+description: Letting candidates choose which references to submit with their application
+date: 2021-07-14
```

Comment:
Use today’s date:

```suggestion
date: 2021-07-19
```

…and update the file name to match: `2021-07-19-selecting-references.md`

Reply:
Done.